packages/ses/src/permits.js

```diff
@@ -1336,7 +1336,7 @@ export const permitted = {
 
   '%IteratorPrototype%': {
     // The %IteratorPrototype% Object
-    '@@iterator': fn,
+    '@@iterator': /** @type {any} */ (fn),
```

Comment:
This property is modified below otherwise it won't pass the check. Editing this is not good but I cannot think of how to do it under the current architecture.

https://github.com/endojs/endo/pull/2383/files#diff-8cc460b0bc4cf783d417fbe07122880176867c0b2c81878afd616f0903e13c56R11